src/org/usfirst/frc2084/CMonster2017/Robot.java~

```diff
@@ -0,0 +1,155 @@
+ * creating this project, you must also update the manifest file in the resource
+ * directory.
+ */
+
+// Note: FRC team 3946 and team 1073 have very good example code on github.
+
+public class Robot extends IterativeRobot {
+
+	Command autonomousCommand;
+	SendableChooser autoChooser; // varaible that allows you to choose between
+									// different autonomouses from the
+									// SmartDashboard
+
+	public static OI oi;
+	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+	public static DriveBase driveBase;
+	public static GearBase gearBase;
+	public static ClimberBase climberBase;
+	public static IntakeBase intakeBase;
+	public static LifterBase lifterBase;
+
+	// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+
+	// autogenerated code declares all the different subsystems
+
+	/**
+	 * This function is run when the robot is first started up and should be
+	 * used for any initialization code.
+	 */
+
+	public void robotInit() {
+		RobotMap.init();
+
+		autoChooser = new SendableChooser();
+		autoChooser.addDefault("MiddleProgram", new MoveForward());
+		autoChooser.addObject("LeftSideAuto", new AutonomousLeftSide());
+		autoChooser.addObject("RightSideAuto", new AutonomousRightSide());
+		SmartDashboard.putData("Autonomous Mode Chooser", autoChooser);
+		// create variable autoChooser and create different instances of the
+		// variable so you
+		// can select different autonomouses through the SmartDashboard
+
+		// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+		driveBase = new DriveBase();
+		gearBase = new GearBase();
+		climberBase = new ClimberBase();
+		intakeBase = new IntakeBase();
+		lifterBase = new LifterBase();
+
+		// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+		// OI must be constructed after subsystems. If the OI creates Commands
+		// (which it very likely will), subsystems are not guaranteed to be
+		// constructed yet. Thus, their requires() statements may grab null
+		// pointers. Bad news. Don't move it.
+		oi = new OI();
+
+		autonomousCommand = new AutonomousCommand();
+
+		// instantiate the command used for the autonomous period
+		// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=AUTONOMOUS
+
+		// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=AUTONOMOUS
+
+		CameraServer.getInstance().startAutomaticCapture();
+		// starts vision stream
+	}
+
+	/**
+	 * This function is called when the disabled button is hit. You can use it
+	 * to reset subsystems before shutting down.
+	 */
+	public void disabledInit() {
+		Robot.driveBase.DisableDriveBase();
+	}
+
+	public void disabledPeriodic() {
+		Scheduler.getInstance().run();
+	}
+
+	public void autonomousInit() {
+		// Scheduler.getInstance().removeAll();
+		// resets distances for the next autonomous command, so values don't
+		// carry over
+		// through different autos
+
+		// schedule the autonomous command (example)
+		if (autonomousCommand != null)
```

Comment:
Get in the habit of using curly braces everywhere you use `if`, `else`, `for`, and `while`